src/08-accessing.js

```diff
@@ -171,7 +191,21 @@ function getFullAddress(person) {}
       numbers: [7185550921, 7185558611],
     };
  */
-function getFlatObject(person) {}
+function getFlatObject(person) {
+  let newObj = {};
+  let newObjTwo;
+
+  newObjTwo = Object.assign(newObj, person.names);
```

Comment:
Nice `Object.assign()` call.  Could you explain how this is working step by step?  Also - can you solve this without using ` Object.assign `?